src/EditorFeatures/Core/FindUsages/IDefinitionsAndReferencesFactory.cs

```diff
@@ -172,6 +175,25 @@ internal static class DefinitionItemExtensions
             return properties;
         }
 
+        private static ImmutableArray<AdditionalProperty> GetDisplayableProperties(ISymbol definition)
+        {
+            var displayableProperties = ImmutableArray<AdditionalProperty>.Empty;
+
+            var containingType = definition.ContainingType;
+            if (containingType != null)
+            {
+                displayableProperties = displayableProperties.Add(new AdditionalProperty(AbstractReferenceFinder.ContainingTypeInfoPropertyName, containingType.Name));
```

Comment:
this is allocating a new array every time you add a property to it.  can you switch to ArrayBuilder instead?